insights/terraform/projects/bfd/api-requests/firehose.tf

```diff
@@ -81,7 +81,10 @@ resource "aws_kinesis_firehose_delivery_stream" "bfd-firehose" {
 # CloudWatch Log Subscription
 resource "aws_cloudwatch_log_subscription_filter" "bfd-access-log-subscription" {
   name            = "${local.full_name}-access-log-subscription"
-  log_group_name  = "/bfd/${local.environment}/bfd-server/access.json"
+  # Set the log group name so that if we use an environment ending in "-dev", it will get logs from
+  # the "real" log group for that environment. So we could make an environment "prod-sbx-dev" that
+  # we can use for development, and it will read from the "prod-sbx" environment.
+  log_group_name  = "/bfd/${replace(local.environment, "-dev", "")}/bfd-server/access.json"
```

Comment:
I'm really interested in this use-case! Maybe there's a better forum to discuss it, but how are you using the `-dev` suffix to facilitate development?

Reply:
It came up also when I was working on the parquet stuff, but basically I want to be able to work on things _without_ also breaking the stuff that already exists. The `test` server is an ok-but-not-great place to test the Firehose / Lambda, and it's been helpful (to me) to attach development assets like `prod-sbx-dev` that ingests real data from `prod-sbx`. This is the only place where that ingestion happens automatically, so with this little change it's easy enough to create a whole BFD Insights environment that piggybacks off of prod-sbx.